src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2214,7 +2215,7 @@ void SessionImpl::populateAdditionalTrackers()
     {
         tracker = tracker.trimmed();
         if (!tracker.isEmpty())
-            m_additionalTrackerList.append({tracker.toString()});
+            m_additionalTrackerList.append({ .url = tracker.toString(), .tier = 0 });
```

Comment:
The default tier is `0`.

Reply:
> The default tier is `0`. It is only explicitly written out with designated initializers now.

What's the reason? The default values were invented to avoid writing them explicitly.

Reply:
Was the previous code unclear for you?

Reply:
>The default values were invented to avoid writing them explicitly.

And what was the reason for `explicit` keyword for constructors? I find them not much different.

>Was the previous code unclear for you?

Yes. I consider using the default initialization form here is bad. I really want to be clear about the tier number. Also the tier is subject to change in a future PR due to `0` is not expected by users.

Reply:
> > The default values were invented to avoid writing them explicitly.
> 
> And what was the reason for `explicit` keyword for constructors? I find them not much different.

I don't see any analogy.

> > Was the previous code unclear for you?
> 
> Yes. I consider using the default initialization form here is bad. I really want to be clear about the tier number. Also the tier is subject to change in a future PR due to `0` is not expected by users.

OK.

Reply:
```suggestion
            m_additionalTrackerList.append({.url = tracker.toString(), .tier = 0});
```